### PR TITLE
Add note about session store's location and development server

### PR DESCRIPTION
**Pull request by limdauto**
## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

## Legal notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -184,6 +184,16 @@ This command will launch a Kedro-Viz server at [localhost:4142](http://localhost
 
 > **Note**: Once the development server is launched at port 4142, the local app will always pull data from that server. To prevent this, you can comment out the proxy setting in `package.json` and restart the dev server at port 4141.
 
+#### Launch the development server with the `SQLiteSessionStore`
+
+Kedro-Viz provides a `SQLiteSessionStore` that users can use in their project to enable experiment tracking functionality. If you want to use this session store with the development server,
+make sure you don't use relative path when specifying the store's location. For example, to specify the local `data` directory within a project as the session store's location, configure the project's `settings.py` as follow:
+
```
Comment (AntonyMilneQB) [inline CONTRIBUTING.md:191]:
```suggestion
Kedro-Viz provides a `SQLiteSessionStore` that users can use in their project to enable experiment tracking functionality. If you want to use this session store with the development server, make sure you don't use a relative path when specifying the store's location. For example, to specify the local `data` directory within a project as the session store's location, configure the project's `settings.py` as follow:
```
Don't know if the line break was intentional?

**Review by AntonyMilneQB — APPROVED** [review]
This looks good, thanks for the fix! I'm just wondering whether `SQLiteSessionStore` should be doing the path resolution itself. Currently it looks like it does
```
@property
    def location(self) -> Path:
        """Returns location of the sqlite_store database"""
        return Path(self._path).expanduser().resolve() / "session_store.db"
```
Should this just be `project_path / Path(self._path)` so that it's always interpreted relative to the project path anyway? Or we don't have `project_path` available in a session store?

**Review by rashidakanchwala — APPROVED** [review]
LGTM! Works well!! :)

**Comment by limdauto** [comment]
@AntonyMilneQB the store's path is project path by default. The thing is we also don't know if user uses relative or absolute path in the session store arg. If it's absolute, the `project_path / self._path` will be broken.

**Comment by AntonyMilneQB** [comment]
> @AntonyMilneQB the store's path is project path by default. The thing is we also don't know if user uses relative or absolute path in the session store arg. If it's absolute, the `project_path / self._path` will be broken.

Are you sure? I think absolute paths will always be resolved rather than failing. e.g. `Path("a/b/c/d/") / Path("/bin") == Path("/bin")`.

Not important but now I'm curious 😀

**Comment by limdauto** [comment]
@AntonyMilneQB as discussed, I have removed the path resolution. Thanks.

**Pull request merged**